packages/origin-247-transfer/src/repositories/EnergyTransferRequestPostgres.repository.ts

```diff
@@ -80,6 +80,7 @@ export class EnergyTransferRequestPostgresRepository implements EnergyTransferRe
                 .createQueryBuilder()
                 .update(EnergyTransferRequestEntity)
                 .set(request.toAttrs())
+                .where({ id: request.id })
```

Comment:
I think that executing UPDATE or DELETE without WHERE or hypothetical "ALL" flag should be prohibited